### PR TITLE
fix: prettier config export

### PR DESCRIPTION
**Pull request by chrstph-dvx**

### CODE REVIEW

File: packages/prettier-config/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+1. Install the peerDependencies of prettier config:
+
+```sh
+npx install-peerdeps -Y --dev @offchainlabs/prettier-config
+```
+
+2. Usage
+
+```ts
```
Comment (spsjvc) [inline packages/prettier-config/README.md:30]: nit: maybe be explicit that this should be done in a Prettier config file, e.g. `prettier.config.js`

File: packages/eslint-config-typescript/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+1. Install the peerDependencies of eslint configs:
+
+```sh
+npx install-peerdeps -Y --dev @offchainlabs/eslint-config-typescript/base
+npx install-peerdeps -Y --dev @offchainlabs/eslint-config-typescript/react
+```
+
+2. Usage
+
+```ts
```
Comment (spsjvc) [inline packages/eslint-config-typescript/README.md:18]: nit: same thing, which file should this go into

File: README.md [hunk 3]
```diff
@@ -5,8 +5,8 @@ Provide reusable typescript and react eslint configs
 1. Install the peerDependencies of eslint configs:
```
Comment (spsjvc) [inline README.md]: nit: we basically duplicate readmes now, should we just link to the individual readmes from here?

File: packages/prettier-config/README.md (hunk 1 above)
Comment (chrstph-dvx) [reply packages/prettier-config/README.md:30]: Update with the different possibilities

File: packages/eslint-config-typescript/README.md (hunk 2 above)
Comment (chrstph-dvx) [reply packages/eslint-config-typescript/README.md:18]: Update with the different possibilities

File: README.md (hunk 3 above)
Comment (chrstph-dvx) [reply README.md]: Make sense

**Review by spsjvc — APPROVED** [review]
LGTM

**Pull request merged**